src/games/strategy/engine/data/AllianceTracker.java

```diff
@@ -34,12 +34,12 @@ public AllianceTracker(final GameData data) {
    *        The alliance to add to.
    */
   protected void addToAlliance(final PlayerID player, final String allianceName) {
-    if (!m_alliances.containsKey(player)) {
+    if (!alliances.containsKey(player)) {
       final Collection<String> alliances = new HashSet<>();
       alliances.add(allianceName);
-      m_alliances.put(player, alliances);
+      this.alliances.put(player, alliances);
```

Comment:
Scope increase, let's keep this to just prefix removal. Converting to a `multimap` would remove the second  `alliances` collection altogether, but that increases the scope yet further.